apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/controllers/fee_test.exs

```diff
@@ -95,26 +135,39 @@ defmodule OMG.ChildChainRPC.Web.Controller.FeeTest do
 
       assert %{
                "success" => true,
-               "data" => [
-                 %{
-                   "amount" => 1,
-                   "currency" => "0x0000000000000000000000000000000000000000",
-                   "subunit_to_unit" => 1_000_000_000_000_000_000,
-                   "pegged_amount" => 1,
-                   "pegged_currency" => "USD",
-                   "pegged_subunit_to_unit" => 100,
-                   "updated_at" => "2019-01-01T10:00:00Z"
-                 },
-                 %{
-                   "amount" => 1,
-                   "currency" => "0x0000000000000000000000000000000000000001",
-                   "subunit_to_unit" => 1_000_000_000_000_000_000,
-                   "pegged_amount" => 1,
-                   "pegged_currency" => "USD",
-                   "pegged_subunit_to_unit" => 100,
-                   "updated_at" => "2019-01-01T10:00:00Z"
-                 }
-               ]
+               "data" => %{
+                 "1" => [
+                   %{
+                     "amount" => 1,
+                     "currency" => "0x0000000000000000000000000000000000000000",
+                     "subunit_to_unit" => 1_000_000_000_000_000_000,
+                     "pegged_amount" => 1,
+                     "pegged_currency" => "USD",
+                     "pegged_subunit_to_unit" => 100,
+                     "updated_at" => "2019-01-01T10:00:00Z"
+                   },
+                   %{
+                     "amount" => 1,
+                     "currency" => "0x0000000000000000000000000000000000000001",
+                     "subunit_to_unit" => 1_000_000_000_000_000_000,
+                     "pegged_amount" => 1,
+                     "pegged_currency" => "USD",
+                     "pegged_subunit_to_unit" => 100,
+                     "updated_at" => "2019-01-01T10:00:00Z"
+                   }
+                 ],
+                 "2" => [
+                   %{
+                     "amount" => 2,
+                     "currency" => "0x0000000000000000000000000000000000000000",
+                     "subunit_to_unit" => 1_000_000_000_000_000_000,
+                     "pegged_amount" => 1,
+                     "pegged_currency" => "USD",
+                     "pegged_subunit_to_unit" => 100,
+                     "updated_at" => "2019-01-01T10:00:00Z"
+                   }
+                 ]
+               }
              } = TestHelper.rpc_call(:post, "/fees.all", missing_param)
```

Comment:
`missing_param` is an empty map which technically is still a parameter. no body test on the other hand would be what the test is describing

Reply:
this probably means that you need to reduce the valid requests to `%{currencies: []}` and reject everything else?